tests/vmi_lifecycle_test.go

```diff
@@ -231,6 +239,9 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			}
 			vmi := libvmi.New(options...)
 
+			if arch == "s390x" {
+				vmi = libvmifact.NewAlpine(options...)
```

Comment:
Again can we document why this required for this test in code comments.

Reply:
review comment not applicable after reverting the changes.